lib/item.rb

```diff
@@ -40,4 +40,13 @@ def total_revenue
       sum + (inv_item.quantity * inv_item.unit_price)
     end
   end
+
+  def total_sold
+    invoice_items = completed_invoice_items
+    invoice_items.collect{|invoice_item| invoice_item.quantity}.reduce(0, :+)
```

Comment:
Prefer `map` over `collect`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.